src/libs/SidebarUtils.ts

```diff
@@ -232,7 +232,7 @@ type ActorDetails = {
  */
 function getOptionData(
     report: Report,
-    reportActions: Record<string, ReportAction[]>,
+    reportActions: Record<string, ReportAction>,
```

Comment:
Has this been wrong til now?

Reply:
yes - `Record<string, ReportAction>` :wink: